src/applications/vaos/services/appointment/transformers.js

```diff
@@ -10,43 +10,19 @@ import {
 } from '../../utils/constants';
 import { transformFacilityV2 } from '../location/transformers';
 
-export function getAppointmentType(appt, useFeSourceOfTruthVA) {
-  // TODO: Update APPOINTMENT_TYPES enum to match API response values.
-  const isCerner = appt?.id?.startsWith('CERN');
-
-  if (useFeSourceOfTruthVA) {
-    if (appt?.type === 'VA') {
-      return APPOINTMENT_TYPES.vaAppointment;
-    }
-
-    if (appt?.type === 'REQUEST') {
-      return APPOINTMENT_TYPES.request;
-    }
-  }
-
-  if (appt?.type === 'COMMUNITY_CARE_APPOINTMENT') {
-    return APPOINTMENT_TYPES.ccAppointment;
-  }
-  if (appt?.type === 'COMMUNITY_CARE_REQUEST') {
-    return APPOINTMENT_TYPES.ccRequest;
+export function getAppointmentType(appt) {
+  if (appt?.type === 'VA') {
+    return APPOINTMENT_TYPES.vaAppointment;
   }
-
-  if (isCerner && isEmpty(appt?.end)) {
+  if (appt?.type === 'REQUEST') {
     return APPOINTMENT_TYPES.request;
   }
-  if (isCerner && !isEmpty(appt?.end)) {
-    return APPOINTMENT_TYPES.vaAppointment;
-  }
-  if (appt?.kind === 'cc' && appt?.start) {
+  if (appt?.type === 'COMMUNITY_CARE_APPOINTMENT') {
     return APPOINTMENT_TYPES.ccAppointment;
   }
-  if (appt?.kind === 'cc' && appt?.requestedPeriods?.length) {
+  if (appt?.type === 'COMMUNITY_CARE_REQUEST') {
     return APPOINTMENT_TYPES.ccRequest;
   }
-  if (appt?.kind !== 'cc' && appt?.requestedPeriods?.length) {
-    return APPOINTMENT_TYPES.request;
-  }
-
   return APPOINTMENT_TYPES.vaAppointment;
```

Comment:
technically, we can never reach this point since vets-api will always return one of the above 4 cases. We can consider adding some kind of warning diagnostic here (maybe in GA?).